algorithms/Frequent_Pattern_Mining/fpm/tg_fpm.gsql

```diff
@@ -0,0 +1,147 @@
+CREATE QUERY tg_fpm(STRING unify_vertex_type, STRING item_vertex_type, STRING connect_edge, DOUBLE support, 
+                    UINT min_length, UINT lim = 10000, INT output_limit = -1, BOOL spit_to_file = FALSE,
```

Comment:
write_to_file is a better name than spit_to_file

Reply:
use `file_path=""` to indicate no file output
we can remove `spit_to_file `

Reply:
updated